source/first_party/optimizers/deep_reaction_optimizer.rst

```diff
@@ -25,8 +25,7 @@ Download and install the plugin from the official GitHub release page listed bel
 
 **Downloads**
 
-- **Plugin (.vip file):**  
-  `CyRxnOpt Plugin – Releases <https://github.com/RxnRover/plugin_dro/releases>`_
+- **Plugin (.vip file):** `DRO Plugin – Releases <https://github.com/RxnRover/plugin_dro/releases>`_
```

Comment:
There is no release for this yet. Honestly, I'd leave this one pointing to the repo, though. Don't bother with a release for now.